Environment/k3d/k8s/persistent-volume.yml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: managed-premium
+  labels:
+    type: local
+spec:
+  storageClassName: managed-premium
+  capacity:
+    storage: 10Gi
```

Comment:
See https://github.com/dolittle/platform-api/pull/36/files#r717593678 about the size